### ISSUE TITLE
Arquivos com as iniciais "Carved-"

### ISSUE THREAD

**Issue by fewirchen** [sepinf-inc/IPED#343]
Por gentileza.

Uma execução do IPED em cima de uma imagem *.dd de um HD com Windows 7 está recuperando praticamente todos os arquivos úteis aos usuários comuns com “Carved-...” no ínício do nome, ou seja, não mostra o nome Windows. A execução é com o parâmetro “carving” habilitado porque se não estiver habilitado não acha estes arquivos.

Qdo monto esta imagem pelo FTK, eu consigo ver todas as pastas do Windows, ou seja, posso afirmar que tá tudo lá, não foram removidos. Porém debaixo de Users, vejo as pastas dos usuários mas quando tento ver o conteúdo, recebo a msg de que não tenho permissão.

Será que é por isso que o IPED está colocando estas iniciais “Carved-...” no ínício do nome? Aí quando o examinador roda o IPED-SearchApp.exe, não consegue saber o nome do arquivo.

Grato.

**Comment by lfcnassif** [comment]
Please post in english. Possibly the File System/partitioning scheme was not recognized by sleuthkit dependency. Please send the full processing log to confirm that.

You could try to create an AD1 logical image with FtkImager, then you can process the AD1 using Iped, so file system structure will be recognized. If needed, It is also possible to include unallocated space in AD1, if you create it per partition.

**Comment by fewirchen** [comment]
Log tem 4MB
[IPED-2021-01-14-23-28-46.log](https://github.com/sepinf-inc/IPED/files/5824838/IPED-2021-01-14-23-28-46.log)

**Comment by fewirchen** [comment]
Sorry, log about 4MB

**Comment by fewirchen** [comment]
This problem processing was the creation in the image was with Linux Caine and execution of IPED in Windows 10. I will prepare the AD1 image with FTK in Windows and protect the evidence by blocking the USB port.

**Comment by lfcnassif** [comment]
> Log tem 4MB
> [IPED-2021-01-14-23-28-46.log](https://github.com/sepinf-inc/IPED/files/5824838/IPED-2021-01-14-23-28-46.log)

Just read the log, not a tool issue. You must pass img_3110.dd image path to -d command line parameter, not its parent folder.

**Comment by fewirchen** [comment]
![20210117_101502_reduzida](https://user-images.githubusercontent.com/77543117/104846782-a88a7000-58b2-11eb-8a45-bbbdb1ec3019.jpg)

Luis Nassif, you were surgical, I pointed to the * .dd image file and right at the beginning of the execution, I could see that it would be different. Thank you very much.

I would like to show a small detail, when I executed passing the parent folder, unfortunately, the image appeared in the right place, showing me wrongly that it was executing, as shown in the figure.

Even so, you solved my problem, one more time, thank you.
